redash/controllers.py

```diff
@@ -450,7 +450,7 @@ def delete(self, visualization_id):
 class QueryResultListAPI(BaseResource):
     @require_permission('execute_query')
     def post(self):
-        params = request.json
```

Comment:
Looks like I'm right:

http://stackoverflow.com/questions/20001229/how-to-get-posted-json-in-flask

I can fix this now if you'd like.